.expeditor/docker-manifest-create.sh

```diff
@@ -10,12 +10,12 @@ function create_and_push_manifest() {
   manifest_tag="${1}"
 
   echo "--- Creating manifest for ${manifest_tag}"
-  docker manifest create "chef/chef:${manifest_tag}" \
-    --amend "chef/chef:${version}-arm64" \
-    --amend "chef/chef:${version}-amd64"
+  docker manifest create "chef/chef-hab:${manifest_tag}" \
+    --amend "chef/chef-hab:${version}-amd64"
+#    --amend "chef/chef-hab:${version}-arm64" # Commenting out the arm64 as we are not supporting it in RC1
```

Comment:
Please note the disabling of the arm64 push in the main description of the PR and also the intended point at which it is planned to be restored.

Reply:
Done.